modules/setting/setting.go

```diff
@@ -774,11 +782,15 @@ func NewContext() {
 
 	InternalToken = loadInternalToken(sec)
 
+	sec = Cfg.Section("storage")
```

Comment:
As said above, I'd have three sections: `[storage.avatars]`, `[storage.attachments]`, `[storage.lfs]`. Or some flexible system like that.